README.md

```diff
@@ -158,15 +158,15 @@ by default. To enable, set the `ENABLE_ANALYTICS` environment variable.
 
 8.  **`/mixins`**: Mixin functions shared through the whole project.
 
-9.  **`/pages`**: This is a starting point because if you want to know what is the website structure, it's the same as this folder's structure. Nuxt reads all the `.vue` files inside this directory and creates the application router based on it. You can find information at [Nuxt's pages directory documentation](https://nuxtjs.org/guide/directory-structure#the-pages-directory). All `.vue` pages prefixed by an underscore are [dynamic routes](https://nuxtjs.org/guide/routing/#dynamic-routes) and we use them to create different pages based on the same template. We also use [nuxt-link](https://nuxtjs.org/guide/routing/) to keep the user inside our webapp router.
+9.  **`/new-content`**: Content for the learning resources cards
```

Comment:
In qiskit.org there is some content generated by writing markdown files with front-matter metadata. Traditionally, we used [frontmatter-markdown-loader](https://github.com/hmsk/frontmatter-markdown-loader) to do it and all the content was in the `content` folder. Now we are transitioning to using [nuxt/content](https://content.nuxtjs.org/) and this is the folder we are using for putting the content load this way. Can you reflect this in the README, please?